src/services/PugQueue.js

```diff
@@ -25,36 +28,37 @@ export class PugQueue {
     this.queue.push(member)
     member.addRole(process.env.PUGS_ROLE)
 
-    return `${String.fromCodePoint(0x2705)} ${name} added to queue. ${this.getQueueState()}`
+    return `${String.fromCodePoint(0x2705)} ${name} ${pug_roles} added to queue. ${this.getQueueState()}`
   }
 
   attemptGameStart (guild) {
     if (this.queue.length < process.env.TEAM_SIZE * 2) {
       return
     }
 
-    this.gameCounter += 1
-    const tempChannelName = `pug-${this.gameCounter}`
-
     let mentions = ``
     this.queue.forEach(member => {
-      mentions += `${member}\n`
-    })
-
-    guild.createChannel(tempChannelName, 'text').then((channel) => {
-      channel.send(`The players for this game are:\n${mentions}`)
-
-      channel.send('Nominate captains and draft teams. This channel will be deleted in an hour.')
-
-      // Start timer to delete channel
-      setTimeout(() => {
-        channel.delete()
-            .then(console.log(`Deleted channel ${tempChannelName}`))
-      }, process.env.TEMP_CHANNEL_LIFETIME)
-
-      channel.guild.channels.get(process.env.PUGS_CHANNEL).send(`Game ready to start, draft teams in channel ${channel}.`)
+      const name = getFullName(member)
+      const pug_roles = getRoles(member)
+      mentions += `${member} ${pug_roles}\n`
     })
 
+    // Announce game start and alert players
+    const announce_channel = guild.client.channels.get(process.env.PUGS_ANNOUNCEMENTCHANNEL)
+    const map = pug_map()
+    announce_channel.send(`❮❮❮\t\t **Match Starting!**\t\t ❯❯❯\n
+*Following players head over to Match Draft:*\n
+${mentions}\n
+
+${map}
+
+❯❯❯\t\t*end of match announcement*\t\t❮❮❮\n
+	`)
+    
+    // Notification for queue full
+    const pugs_channel = guild.client.channels.get(process.env.PUGS_CHANNEL)
+    pugs_channel.send(`We have our 12 players, lets go!`)
```

Comment:
Maybe mention the announcements channel in this message to people can easily check if they are in the game.

Reply:
don't think its really needed. if you click the notification that you receive it takes you to that channel anyway.